pep_sphinx_extensions/pep_theme/static/style.css

```
@@ -225,14 +226,14 @@ div.table-wrapper {
 table {
     width: 100%;
     border-collapse: collapse;
-    border-top: 1px solid var(--colour-background-accent);
-    border-bottom: 1px solid var(--colour-background-accent);
+    border-top: 1px solid var(--colour-background-accent-light);
+    border-bottom: 1px solid var(--colour-background-accent-light);
```

```suggestion
    border-top: 1px solid var(--colour-background-accent-strong);
    border-bottom: 1px solid var(--colour-background-accent-strong);
```

To be consistent with the current and the lines